jobs/awslogs/templates/bin/pre-start

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+echo "[general]"
+echo "state_file = /var/vcap/sys/log/awslogs/agent-state"
+
+for SYSTEM_LOG in /var/log/audit/audit.log /var/log/auth.log /var/log/dpkg.log /var/log/messages /var/log/syslog
```

Comment:
Ah, I was thinking we'd write out the config for auto-discovered files next to the static config from the release, so we can still expose basic configuration through the job properties / erb templates but we aren't really doing that today, so I'll defer to your judgement.

Either way, the job spec file will need to be updated to reference this pre-start script (and possible remove the reference to the existing config file template)